web/src/beta/features/Editor/Map/InspectorPanel/LayerInspector/DataSource/index.tsx

```diff
@@ -47,6 +65,27 @@ const DataSource: FC<Props> = ({ selectedLayer }) => {
     [saveCollapseState]
   );
 
+  const handleTitleUpdate = useCallback(() => {
+    if (!localTitle || localTitle === selectedLayer.title) return;
+    onLayerNameUpdate?.({ layerId: selectedLayer.id, name: localTitle });
+  }, [localTitle, onLayerNameUpdate, selectedLayer.id, selectedLayer.title]);
+
+  const handleLayerUrlUpdate = useCallback(
+    (url: string) => {
+      const type = selectedLayer.config.data.type;
+      onLayerConfigUpdate?.({
+        layerId: selectedLayer.id,
+        config: {
+          data: {
+            type,
+            url
+          }
+        }
+      });
```

Comment:
```suggestion
      onLayerConfigUpdate?.({
        layerId: selectedLayer.id,
        config: {
          data: {
            ...selectedLayer.config.data,
            url
          }
        }
      });
```
Maybe better like this?